<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControllerWidget</class>
 <widget class="QWidget" name="ControllerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0" colspan="2">
    <widget class="QPushButton" name="m_colorButton">
     <property name="text">
      <string>Select Color</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSlider" name="m_transparencySlider">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="m_opacityReturnDefaultButton">
     <property name="text">
      <string>Default Opacity</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="m_colorReturnDefaultButton">
     <property name="text">
      <string>Default Color</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QPushButton" name="m_ShowLinePushButton">
     <property name="text">
      <string>Show/Hide Electrode Line</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
